.circleci/unittest/linux/scripts/environment.yml

```diff
@@ -6,6 +6,7 @@ dependencies:
   - pytest-cov
   - codecov
   - pip
+  - libpng
```

Comment:
I think the same is needed for Windows in https://github.com/pytorch/vision/blob/master/.circleci/unittest/windows/scripts/environment.yml ? Not sure why tests are not failing there